scripts/node_suite_run.py

```
@@ -0,0 +1,111 @@
+    if not os.path.isdir(md):
+        continue
+    for dp, _, files in os.walk(md):
+        for f in files:
+            if f.endswith(".ts") and not f.endswith(".d.ts"):
+                tests.append((mod, os.path.join(dp, f)))
+
+
+def run_one(args):
+    mod, path = args
+    try:
+        n = subprocess.run([NODE, path], capture_output=True, text=True, timeout=30)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)